virtcontainers/qemu.go

```diff
@@ -1550,7 +1567,12 @@ func (q *qemu) hotplugAddMemory(memDev *memoryDevice) (int, error) {
 		memDev.slot = maxSlot + 1
 	}
 
-	share, target, memoryBack := q.getMemArgs()
+	share, target, memoryBack, err := q.getMemArgs()
+	if err != nil {
+		q.Logger().WithError(err).Error("hotplug memory")
```

Comment:
is this log for debugging? if not, why it was not added in the previous call to `q.getMemArgs()`

Reply:
yes, it's for debugging.
Removed it.